src/FactSystem/FactMetaData.h

```diff
@@ -121,6 +121,7 @@ class FactMetaData : public QObject
     static const int defaultDecimalPlaces = 3;
 
     static ValueType_t stringToType(const QString& typeString, bool& unknownType);
+    static size_t typeToSize(ValueType_t);
```

Comment:
Can you add type here? I know nitpick, but sometimes people use this as an indication of an unused param. So it could be confusing.